tests/integration/test-app/Gemfile.lock

```diff
@@ -52,16 +52,13 @@ GEM
       mime-types (>= 1.16, < 4.0)
       netrc (~> 0.8)
     ruby2_keywords (0.0.5)
-    sinatra (3.0.5)
-      mustermann (~> 3.0)
-      rack (~> 2.2, >= 2.2.4)
-      rack-protection (= 3.0.5)
-      tilt (~> 2.0)
+    roda (3.68.0)
+      rack (~> 3.0.6)
     table_print (1.5.7)
     thin (1.8.1)
       daemons (~> 1.0, >= 1.0.9)
       eventmachine (~> 1.0, >= 1.0.4)
-      rack (>= 1, < 3)
+      rack (~> 3.0.6)
```

Comment:
Updating the Gemfile.lock manually can cause issues - I noticed this broke the conjur buildpack when I tried to pin the `thin` dependency to a higher rack version.

You should probably use the Gemfile.lock that is generated via a `bundle install`. You can use the command below if you don't have the proper ruby version locally:
`rm Gemfile.lock && docker run -v $PWD:/src -w /src ruby:3.2 sh -c "bundle install"`

Reply:
I was able to fix the Gemfile & Gemfile.lock in tests/integration/test-app using this method, but ./Gemfile.lock & ./Gemfile (in the root directory) is a bit more complicated as it depends on actionpack which I cannot find anywhere in the project.

Here is the error produced when I tried it for those files:
`Because actionpack >= 7.0.0.alpha1, < 7.0.5 depends on rack >= 2.2.0, < 3.A
  and actionpack >= 6.1.0.rc1, < 7.0.0.alpha1 depends on rack >= 2.0.9, < 3.A,
  actionpack >= 6.1.0.rc1, < 7.0.5 requires rack >= 2.0.9, < 3.A.
And because actionpack >= 7.0.5 depends on rack >= 2.2.4, < 3.A
  and rspec-rails >= 6.0.0.rc1 depends on actionpack >= 6.1,
  rspec-rails >= 6.0.0.rc1 requires rack >= 2.0.9, < 3.A.
So, because Gemfile depends on rack >= 3.0.6
  and Gemfile depends on rspec-rails ~> 6.0,
  version solving has failed.`